src/containers/shared/components/TokenTableRow.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export interface TokenTableRowProps {
+  shouldDisplay?: boolean
+  label: string
+  value: any
+}
+
+export const TokenTableRow = (props: TokenTableRowProps) => {
+  const { shouldDisplay = true, label, value } = props
+  return (
+    <>
+      {shouldDisplay && (
+        <tr className="row">
+          <td className="col1">{label}</td>
+          <td className="col2">{value}</td>
+        </tr>
```

Comment:
I think this is the same as `SimpleRow` for transactions. Perhaps those should be merged into something that's just `Row` or `TableRow` or something.

Reply:
Not sure, the token-table  is using `<table>` tag, I assume  refactoring the entire token table would be need if we want to use the SimpleRow

Reply:
Oh, good point - nvm